packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -323,6 +337,13 @@ Resources:
     Properties:
       AssumeRolePolicyDocument:
         Statement:
+          # allows this role to be assumed with session tags.
+          # see https://docs.aws.amazon.com/IAM/latest/UserGuide/id_session-tags.html#id_session-tags_permissions-required
+          - Action: sts:TagSession
```

Comment:
I'm surprised that this appears in the `AssumeRolePolicyDocument`.

But you've tested this, so this is definitely correct, right?

Reply:
I was also surprised but yes.